packages/beacon-node/src/network/peers/peersData.ts

```diff
@@ -38,8 +38,8 @@ export class PeersData {
     return this.connectedPeers.get(peerIdStr)?.agentVersion ?? "NA";
   }
 
-  getPeerKind(peerIdStr: string): ClientKind {
-    return this.connectedPeers.get(peerIdStr)?.agentClient ?? ClientKind.Unknown;
+  getPeerKind(peerIdStr: string): ClientKind | null {
+    return this.connectedPeers.get(peerIdStr)?.agentClient ?? null;
```

Comment:
No need to use `Nullish coalescing operator` with null value?

Reply:
need it to return `ClientKind | null` vs `ClientKind | null | undefined`